### PR TITLE
[CALCITE-4901] RelToSqlConverter adds redundant order by output column for the result sql

### PR DESCRIPTION
**Pull request by hannerwang**
…n for the result sql

### CODE REVIEW

**Review by NobiGo — APPROVED** [review]
After reformat the Sql , Others is good to me.

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java [hunk 1]
```diff
@@ -1545,7 +1556,7 @@ private void checkHavingAliasSameAsColumn(boolean upperAlias) {
     String query = "select \"product_id\" from \"product\" "
         + "order by \"net_weight\" asc, \"gross_weight\" desc, \"low_fat\"";
     final String expected = "SELECT"
-        + " \"product_id\", \"net_weight\", \"gross_weight\", \"low_fat\"\n"
+        + " \"product_id\"\n"
```
[the remark this answers is not in the document]
Comment (hannerwang) [reply core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java]: Fixed, Thanks.

**Review by NobiGo — APPROVED** [review]
LGTM

**Comment by hannerwang** [comment]
The Druid tests failed because of timeout, I don't know how to trigger it rerun, could any one gives advice to resolve it?

**Comment by NobiGo** [comment]
> The Druid tests failed because of timeout, I don't know how to trigger it rerun, could any one gives advice to resolve it?

According to https://calcite.apache.org/develop/#contributing

In the special case, that the CI build failed, and the failure is not caused by your changes create an empty commit (git commit --allow-empty) and push it.

**Comment by hannerwang** [comment]
> > The Druid tests failed because of timeout, I don't know how to trigger it rerun, could any one gives advice to resolve it?
> 
> According to https://calcite.apache.org/develop/#contributing
> 
> In the special case, that the CI build failed, and the failure is not caused by your changes create an empty commit (git commit --allow-empty) and push it.

Thanks,  Fixed.

**Pull request closed without merging**